src/libraries/System.Private.CoreLib/src/System/Resources/RuntimeResourceSet.cs

```diff
@@ -163,7 +163,7 @@ namespace System.Resources
     // resource files containing thousands of resources.
     //
 #if CORERT
-    public  // On CoreRT, this must be public because of need to whitelist past the ReflectionBlock.
+    public  // On CoreRT, this must be public because of need to allowlist past the ReflectionBlock.
```

Comment:
This comment is a reference to a .NET Native list. We don't care about that list anymore.

```suggestion
    public  // On CoreRT, this must be public to prevent it from getting reflection blocked.
```